k8s/api-deployment.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    kompose.cmd: kompose convert
+    kompose.version: 1.17.0 (HEAD)
```

Comment:
Do we need this annotation ?

Reply:
Nope, autogenerated from kustomize and can be removed